python-sdk/src/astro/databricks/delta.py

```diff
@@ -24,11 +24,15 @@
 
 
 class DeltaDatabase(BaseDatabase):
+
+    LOAD_OPTIONS_CLASS_NAME = "DeltaLoadOptions"
+
     _create_table_statement: str = "CREATE TABLE IF NOT EXISTS {} USING DELTA AS {} "
 
-    def __init__(self, conn_id: str, table: BaseTable | None = None):
+    def __init__(self, conn_id: str, table: BaseTable | None = None, load_options: LoadOptions | None = None):
         super().__init__(conn_id)
         self.table = table
+        self.load_options = load_options or DeltaLoadOptions.get_default_delta_options()
```

Comment:
@tatiana I think this is why utkarsh was able to get rid of the lines we asked about here https://github.com/astronomer/astro-sdk/pull/1540/files#r1063468038.

Reply:
Yup, correct.